src/VisualStudio/Core/Def/Implementation/ProjectSystem/IVisualStudioHostDocument.cs

```diff
@@ -97,6 +97,12 @@ internal interface IVisualStudioHostDocument : IDisposable
         uint GetItemId();
 
         /// <summary>
+        /// <see cref="IVsHierarchy"/> of the shared (or project k) project this <see cref="IVisualStudioHostDocument"/> belongs to.
+        /// it will return null if it is normal document.
+        /// </summary>
+        IVsHierarchy SharedHierarchy { get; }
```

Comment:
Do you have something in mind?


-          heejae

From: Kevin Pilch-Bisson [mailto:notifications@github.com]
Sent: Thursday, April 2, 2015 7:58 AM
To: dotnet/roslyn
Cc: HeeJae Chang
Subject: Re: [roslyn] report project k/shared project IVsHierarchy to error list (#1730)


In src/VisualStudio/Core/Def/Implementation/ProjectSystem/IVisualStudioHostDocument.cs<https://github.com/dotnet/roslyn/pull/1730#discussion_r27664445>:

> @@ -97,6 +97,12 @@ internal interface IVisualStudioHostDocument : IDisposable

>          uint GetItemId();

>

>          /// <summary>

> +        /// <see cref="IVsHierarchy"/> of the shared (or project k) project this <see cref="IVisualStudioHostDocument"/> belongs to.

> +        /// it will return null if it is normal document.

> +        /// </summary>

> +        IVsHierarchy SharedHierarchy { get; }

Let's think of a better xml doc comment here.

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/1730/files#r27664445>.